src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubMediator.cs

```diff
@@ -220,10 +220,10 @@ public DistributedPubSubMediator(DistributedPubSubSettings settings)
             {
                 if (_registry.TryGetValue(_cluster.SelfAddress, out var bucket))
                 {
-                    if (bucket.Content.TryGetValue(remove.Path, out var valueHolder) && valueHolder.Ref != null)
+                    if (bucket.Content.TryGetValue(remove.Path, out var valueHolder) && !Equals(valueHolder.Ref, ActorRefs.Nobody))
```

Comment:
Use the `IsNobody` extension method here IMHO. It'll perform a null check and this reference comparison to `ActorRefs.Nobody` as well.